src/structures/User.js

```diff
@@ -91,9 +92,14 @@ class User extends Base {
        * @type {?UserFlags}
        */
       this.flags = new UserFlags(data.public_flags);
+      
     }
   }
 
+  get asMention(){
+    return `<@${this.id}>`;
+  }
```

Comment:
Isn't this the exact same as `User#toString()`?

Reply:
wait, toString already do that?

Reply:
i thougth it would return the object as string

Reply:
Yes, it does.

Reply:
https://github.com/discordjs/discord.js/blob/5ca97c93515d4dfaa2b4951a020abc000115ed4d/src/structures/User.js#L248

Reply:
i mean, return a json

Reply:
but anyway

Reply:
thx for your attention